Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift

```diff
@@ -76,13 +76,14 @@ extension AirRobeEndpoint {
             "anonymous_id": UIDevice.current.identifierForVendor?.uuidString ?? "",
             "session_id": sessionId,
             "event_name": eventName,
-            "properties": properties
+            "properties": properties,
+            "created_at": Date().iso8601withFractionalSeconds
         ]
         return AirRobeEndpoint(
-            method: .POST,
-            path: "/telemetry_events",
+            method: .PUT,
+            path: "/v1",
             requestBody: requestBody,
-            host: configuration?.mode == .production ? AirRobeHost.airRobeConnectorProduction.rawValue : AirRobeHost.airRobeConnectorSandbox.rawValue
+            host: telemetryEventHost
```

Comment:
I forgot to let you know that we currently don't have a telemetry endpoint for staging. I will create one for staging but I guess it's not a priority as TI probably won't be updating the version of the SDK anytime soon.

Actually, just use this domain for staging: `http://stg.events.airdemo.link` save you updating it later on. This is what the domain name will be when I create it.

Reply:
@johnAirRobe I've updated `sandbox` endpoint as `http://events.stg.airdemo.link` but it gave me an error that the hostname could not be found.
Is this intended behavior or anything went wrong?

Reply:
I have yet to set up the infra to support this yet but will do soon.